Assets/HoloToolkit/Build/Editor/BuildDeployPortal.cs

```diff
@@ -170,7 +170,7 @@ public static AppDetails QueryAppDetails(string baseAppName, ConnectInfo connect
                 AppList appList = JsonUtility.FromJson<AppList>(appListJSON);
                 for (int i = 0; i < appList.InstalledPackages.Length; ++i)
                 {
-                    string appName = appList.InstalledPackages[i].PackageFullName;
+                    string appName = appList.InstalledPackages[i].Name;
```

Comment:
This introduces ambiguity that will bite people. PFN should remain and propagate all the way out. Require callers and users to use PFN.